internal/database/migrations/51_add_project_column.up.sql

```
@@ -0,0 +1,150 @@
+create extension if not exists ltree;
+
+-- Change the 'name' column of the projects table to use the 'ltree' type. It wil contain the full name of the project.
+alter table projects
+  alter column name drop default,
+  alter column name type ltree using name::ltree;
+
+-- Add the 'project' column to the projects table, using the 'ltree' type. It will contain the full name of the parent
+-- project. We add it as nullable first, backfill existing rows, and then enforce NOT NULL so the migration works on
+-- populated tables.
+alter table projects add column project ltree;
+update projects set project = ''::ltree where project is null;
```

SQLFluff (4.2.1)

[error] The 'where' keyword should always start a new line.

(LT14)